windows/hid.c

```diff
@@ -818,6 +822,55 @@ int HID_API_EXPORT HID_API_CALL hid_get_feature_report(hid_device *dev, unsigned
 #endif
 }
 
+
+int HID_API_EXPORT HID_API_CALL hid_get_input_report(hid_device *dev, unsigned char *data, size_t length)
+{
+#if 0
```

Comment:
Does it make sense to keep this disabled code? I know it is the same way in `hid_get_feature_report()`, but what purpose does this code section serve?

Reply:
hid_get_feature_report had it so I included it as well. The only benefit I can think of is that it shows that the other way is known and wrong, even though it is much shorter.